### ISSUE TITLE
How do I change to dark mode?

### ISSUE THREAD

**Issue by Simao-Santos** [juliagoda/antimicroX#140]
This is a noobish question, but I can't find how to change the application to dark mode.  Can someone help?

![image](https://user-images.githubusercontent.com/26836021/86498827-85b5c580-bd7f-11ea-9d03-726b5141e6a9.png)

I provided a screenshot of the settings menu that I'm shown.

**Comment by juliagoda** [comment]
It's not there yet, but it will be soon.

**Comment by Simao-Santos** [comment]
Ah okay! Since I saw it in the README screenshot, I thought it was already up and running.
Thank you for this awesome project!

**Comment by Simao-Santos** [comment]
@juliagoda I also had a question regarding the translations.
I don't know if you prefer that I open a new issue or can just ask here.

**Comment by juliagoda** [comment]
Maybe another issue for other newcomers and not only. It will be easy to find it later

**Comment by pktiuk** [comment]
Most of the QT based apps have already implemented theming (I assume that if you want a dark theme, you have it enabled system-wide).

@Simao-Santos 
If you have a dark theme on your Linux you can force most of Qt-based apps to use system-theme.
Check this [thread](https://askubuntu.com/questions/706528/qt-apps-stopped-inheriting-gtk-themes)
To sum up, in my case it was:
- installing `qt5-style-plugins`
- adding line `QT_QPA_PLATFORMTHEME=gtk2` to file `~/.profile` or `/etc/environment` (second one requires sudo)

@juliagoda 
Maybe we should add somewhere information about theming options?

**Comment by juliagoda** [comment]
I don't see a problem to add in settings tab "Styles" anyway and then give here option according to dark icon. Many applications have such choices. It's not difficult to use Qt Styles in code

**Comment by sirlucjan** [comment]
I have an irresistible impression that someone here is trying to force an open door. The program is designed not only for users who are familiar with the system but also for beginners. If someone wants to use the settings that @pktiuk offers, they will use them, but most users will prefer to click the appropriate settings. If you feel like you can use it yourself, it is great, but don't expect it from others. This program is for users, not users for the program. Isn't it better to take care of something useful rather than look for problems where they don't exist? It's the way nowhere.

**Comment by juliagoda** [comment]
> I have an irresistible impression that someone here is trying to force an open door. The program is designed not only for users who are familiar with the system but also for beginners. If someone wants to use the settings that @pktiuk offers, they will use them, but most users will prefer to click the appropriate settings. If you feel like you can use it yourself, it is great, but don't expect it from others. This program is for users, not users for the program. Isn't it better to take care of something useful rather than look for problems where they don't exist? It's the way nowhere. 

Of course, we can describe such steps (maybe someone will like the methods), but if there exist shorter and more intiuitive methods (for users of course), then that should be added as well.

**Comment by pktiuk** [comment]
I think the best solution is just giving freedom of choice for users. For now, we have only that Qt-based solution, but it would be nice to add an optional dark theme in settings (i think it would be also the most viable solution for windows users).
@juliagoda 
This is "workaround", because we as developers haven't dealt with this issue yet. We could possibly achieve the same result  by editing `.desktop` file.

@sirlucjan 
I don't want to force an open door.  I am just saying, that we should think about using system-wide themes because it is already implemented.
